packages/preprocess/src/process-queries.cjs

```diff
@@ -261,6 +267,15 @@ const createDefaultProps = function (filename, componentDevelopmentMode, duckdbQ
 			}, 5000);
 		};
 
+		const longQueryNotifier = (info) => {
+			toasts.add({
+				id: Math.random(),
```

Comment:
is this what we want? Any issue with passing a random id?

Reply:
Shouldn't be an issue, just meant to have some sort of differentiator between toasts